backend/fms_core/template_importer/importers/library_qc.py

```diff
@@ -0,0 +1,104 @@
+import copy
+from fms_core.template_importer.row_handlers.library_qc.library_qc import LibraryQCRowHandler
+
+from fms_core.models import PropertyType, Protocol, Process
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.library_preparation import LibraryRowHandler, LibraryBatchRowHandler
+from fms_core.templates import LIBRARY_QC_TEMPLATE
+from collections import defaultdict
+from .._utils import float_to_decimal_and_none, input_to_date_and_none
+
+PROPERTIES_STARTING_INDEX = 5 #TODO verify that this index is correct
+
+# {{TEMPLATE PROPERTY NAME : DB PROPERTY NAME}
+TEMPLATE_PROPERTY_MAPPING = {
+    "Measured Volume (uL)": "Library Measured Volume",
+    "Volume Used (uL)": "Library Volume Used",
```

Comment:
Volume used is stored at the processMeasurement level (it is duplicating information to store it too). This should not be a property. On the other hand you are not storing the value of the library size which is not stored in a permanent storage. You should add the library size here instead. We save the library size in the library model but a subsequent QC would override the value and the historical value would not be available to users anymore.

Reply:
Okay. I don't see a property type that I could use for that though. Should I add a new "Library Size" property in my migration?

Reply:
You create a new one in the migration yeah.